utils/scripts/generate-github-action-file.py

```diff
@@ -216,6 +217,8 @@ def add_main_job(i, workflow, needs, scenarios):
 
         if scenario == "TRACE_PROPAGATION_STYLE_W3C":  # TODO: fix weblog to allow this value for old tracer
             step["if"] = "${{ matrix.variant.library != 'python' }}"  # TODO
+        elif scenario == "APPSEC_BLOCKING":
+            step["if"] = "${{ matrix.library == 'java' }}"
```

Comment:
Why? if the test if flagged as fail, it's ok to run it.

Reply:
We can run it, but we're gonna waste CPU because it won't pass outside java